Makefile.setup

```diff
@@ -128,7 +128,7 @@ install-python-deps::
 	command -v ${PIP} >/dev/null 2>&1
 	${PIP} install ${PIP_INSTALL_ARGS} \
 		click \
-		conan \
+		"conan==1.59.0" \
```

Comment:
Please use `conan<2.0` instead. There will no doubt be further bug fixes for the `1.x` Conan.

Reply:
Changed everything to conan<2.0.0